### ISSUE TITLE
Cannot apt-get update

### ISSUE THREAD

**Issue by tienhuynh5312** [cilix-lab/ubuntu-wrt#6]
After install ubuntuwrt, i ran `apt-get update`, and i had the following problems. Internet works just fine.
```
Err:1 http://wrt.hinrichs.io/ubuntu xenial-wrt InRelease
  Temporary failure resolving 'wrt.hinrichs.io'
Err:2 http://ports.ubuntu.com/ubuntu-ports xenial InRelease
  Temporary failure resolving 'ports.ubuntu.com'
Err:3 http://wrt.hinrichs.io/ubuntu zesty InRelease
  Temporary failure resolving 'wrt.hinrichs.io'
Reading package lists... Done
W: Failed to fetch http://ports.ubuntu.com/ubuntu-ports/dists/xenial/InRelease  Temporary failure resolving 'ports.ubuntu.com'
W: Failed to fetch http://wrt.hinrichs.io/ubuntu/dists/xenial-wrt/InRelease  Temporary failure resolving 'wrt.hinrichs.io'
W: Failed to fetch http://wrt.hinrichs.io/ubuntu/dists/zesty/InRelease  Temporary failure resolving 'wrt.hinrichs.io'
W: Some index files failed to download. They have been ignored, or old ones used instead.
```

**Comment by cilix-lab** [comment]
Is WAN setup? According to that, the domains are not resolving, which could mean internet is not working or DNS is not working.  

Try pinging a domain and an IP like: `ping -c3 google.com` and `ping -c3 8.8.8.8` and see what happens. What kind of internet service do you have? Does it work at all on clients?  

Check if BIND9 is working: `systemctl status bind9` and check `resolv.conf` like: `cat /etc/resolv.conf`.

**Comment by tienhuynh5312** [comment]
I have Comcast Xfinity Cable Internet. The router is behind one DOCSIS 3.0 cable gateway. It is very strange that all the clients was having the internet, and then there was no internet connection after a couple reboots. Because of that reason, i was thinking the wan was setup automatically just like openwrt or ddwrt works right away after installing them.

**Comment by cilix-lab** [comment]
Yes, that's because they all (including UbuntuWRT) are setup for DHCP, so it just works. From what you are saying, it seems it might be a DNS problem. Maybe BIND stopped working... as a workaround, you can setup manual DNS on your clients or change this line in `/etc/dhcp/dhcpd.conf`: `option domain-name-servers 192.168.1.1;` to `option domain-name-servers 8.8.8.8, 8.8.4.4;` (if you're ok using Google's 8.8.8.8 and 8.8.4.4), but to figure out what happened, you should try the commands I posted on the router and check BIND9 status.  

Post your results to shed some light on this issue.

**Comment by tienhuynh5312** [comment]
Hi cilix, thank you for your quick reply.
Everything works right out of the box after I reinstalled it again no further configuration needed to run `apt update`.
Before that what i did is I plugged the usb right before i flashed the firmware from Linksys page. And the reboot.
However the "right way" is to not plugging in just let it flashed, then reboot completely, shut it down, plug in usb and turn it on.
